core/domain/user_jobs_continuous_test.py

```diff
@@ -929,102 +766,41 @@ def test_scaler_multiplier_independence(self):
-        # is below the minimum), so the expected impact score is just the
-        # impact score for exploration 1.
-        expected_user_impact_score = round(
-            math.log(self.MIN_NUM_COMPLETIONS) *
-            (self.ABOVE_MIN_RATING - self.MIN_AVERAGE_RATING) *
-            self.MULTIPLIER)
-        # Verify that the impact score matches the expected.
+        exploration = self._create_exploration(self.EXP_ID_3, user_a_id)
+        self._rate_exploration(exploration.id, 5, 3)
         self._run_computation()
         user_stats_model = user_models.UserStatsModel.get(user_a_id)
-        self.assertEqual(
-            user_stats_model.impact_score, expected_user_impact_score)
+        self.assertEqual(user_stats_model.impact_score, 0)
 
-    def test_num_completions_independence(self):
-        """Test that when one exploration has less than the minimum number
-        of completions, the other exploration's impact score is not impacted.
+    def test_impact_for_exp_with_no_ratings(self):
+        """Test that when an exploration has no ratings, the impact returned
+        from the impact function is 0.
         """
-        # Sign up a user and have them create two explorations.
+        # Sign up a user and have them create an exploration.
         user_a_id = self._sign_up_user(
             self.USER_A_EMAIL, self.USER_A_USERNAME)
-        exploration_1 = self._create_exploration(self.EXP_ID_1, user_a_id)
-        exploration_2 = self._create_exploration(self.EXP_ID_2, user_a_id)
-        # Give these explorations as many ratings as necessary to avoid
-        # the scaler for number of ratings.
-        self._rate_exploration(
-            exploration_1.id, self.NUM_RATINGS_SCALER_CUTOFF,
-            self.ABOVE_MIN_RATING)
-        self._rate_exploration(
-            exploration_2.id, self.NUM_RATINGS_SCALER_CUTOFF,
-            self.ABOVE_MIN_RATING)
-        # Give one exploration the minimum number of completions. Give the other
-        # only one.
-        self._complete_exploration(exploration_1, self.MIN_NUM_COMPLETIONS)
-        self._complete_exploration(exploration_2, 1)
-        # We expect the second exploration to yield 0 (since its average rating
-        # is below the minimum), so the expected impact score is just the
-        # impact score for exploration 1.
-        expected_user_impact_score = round(
-            math.log(self.MIN_NUM_COMPLETIONS) *
-            (self.ABOVE_MIN_RATING - self.MIN_AVERAGE_RATING) *
-            self.MULTIPLIER)
-        # Verify that the impact score matches the expected.
-        self._run_computation()
-        user_stats_model = user_models.UserStatsModel.get(user_a_id)
-        self.assertEqual(
-            user_stats_model.impact_score, expected_user_impact_score)
+        self._create_exploration(self.EXP_ID_1, user_a_id)
+        user_stats_model = user_models.UserStatsModel.get(user_a_id,
```

Comment:
Yes, I think so. The tests should be blind to specific details of the implementation.